src/lib/objects.js

```diff
@@ -0,0 +1,70 @@
+const createPerson = (name, age) => {
+  return {
+    name,
+    age,
+  };
+};
+
+const getName = object => {
+  return object.name;
+};
+
+const getProperty = (property, object) => {
+  return object[property];
+};
+
+const hasProperty = (property, object) => {
+  if (object[property]) {
+    return true;
+  }
+  return false;
+};
+
+const isOver65 = person => {
+  if (person.age > 65) {
+    return true;
+  }
+  return false;
```

Comment:
As above, you could simply return the result of evaluating the condition since you are only looking for a boolean value here:
```
  return person.age > 65;
```